pkg/controller/controllercmd/builder.go

```diff
@@ -304,6 +314,17 @@ func (b *ControllerBuilder) Run(ctx context.Context, config *unstructured.Unstru
 		return nil
 	}
 
+	if b.canUseSNOLeaderElection {
```

Comment:
turning this into `if !b,userExplicitlySetLeaderElectionValues`

Reply:
updated if statement